src/Components/App/App.js

```diff
@@ -24,6 +25,14 @@ function App() {
     .catch(err => console.log(`There has been an error: ${err}`))
   }, [dispatch]);
 
+  useEffect(() => {
+    getFriends(1)
+      .then(data => {
+        console.log('friends', data.data.friends);
+      })
+      .catch(err => console.log(`There has been an error: ${err}`))
+  }, []);
+
```

Comment:
Actually ignore my last comment - I think this code block should be removed entirely, right? It's not actually doing anything and lives in `App.js`.

@Gderibeaux It seems like maybe you were using this in `App.js` at first to get it working, then moved to the `FriendsPage` component but forgot to delete your code? Can you confirm?